### PR TITLE
Update add-on description to be in line with other add-ons

### PR DESCRIPTION
**Pull request by CodeFinder2**
This slightly changes the subtitle as displayed in the Supervisor's add-on page. According to other add-ons, it should briefly explain what the add-on does.

#### Checklist
- [x] PR done against `dev` branch
- [ ] Updated `common/Dockerfile` with new version
- [ ] Updated `zigbee2mqtt/CHANGELOG.md` file with new changes
- [ ] Updated `zigbee2mqtt/DOCS.md` with any changes
- [ ] Add any new configuration options to `zigbee2mqtt/config.json` and `zigbee2mqtt-edge/config.json`.

### CODE REVIEW

File: zigbee2mqtt/config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "Zigbee2mqtt",
   "version": "1.18.3-1",
   "slug": "zigbee2mqtt",
-  "description": "Zigbee2mqtt add-on",
+  "description": "Use your Zigbee devices without the vendors bridge or gateway",
```
Comment (afoeder) [inline zigbee2mqtt/config.json:5]: * ZigBee
* vendor's

**Review by afoeder — CHANGES_REQUESTED** [review]
Hi, please fix the typos, otherwise it's good to go for me 👍

File: zigbee2mqtt/config.json (hunk 1 above)
Comment (CodeFinder2) [reply zigbee2mqtt/config.json]: Thanks for your suggestions, done! :-)

**Review by afoeder — APPROVED** [review]

**Comment by CodeFinder2** [comment]
?

**Comment by CodeFinder2** [comment]
Why has this been closed without any explanation / merge? @Koenkk

**Comment by Koenkk** [comment]
@CodeFinder2 sorry, seems this was done automatically on the 1.19.1-1 release. @ciotlosm can you review this mr?

**Comment by ciotlosm** [comment]
Closing and re-opening to re-trigger checks.

**Review by ciotlosm — APPROVED** [review]

**Pull request merged**